### PR TITLE
Add python docker example

### PR DESCRIPTION
**Pull request by abhishekamralkar**
A Docker example in Python.

### CODE REVIEW

File: examples/python-docker/README.org [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#+title: Python-Docker
```
Comment (anubhavmishra) [inline examples/python-docker/README.org:1]: I would use the [markdown](https://en.wikipedia.org/wiki/Markdown) format for the README.

Comment (anubhavmishra) [reply examples/python-docker/README.org:1]: Also, please follow the guide shown [here](https://github.com/hashicorp/terraform-cdk/tree/master/examples/python-aws#python-aws) and update the README with the instructions accordingly.

File: examples/python-docker/Pipfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+constructs="3.0.4"
+cdktf="0.0.12"
```
Comment (anubhavmishra) [inline examples/python-docker/Pipfile:11]: I would use the latest version of cdktf package.

File: examples/python-docker/README.org (hunk 1 above)
Comment (abhishekamralkar) [reply examples/python-docker/README.org]: README updated as per the guide and formatting changed to markdown.

File: examples/python-docker/Pipfile (hunk 2 above)
Comment (abhishekamralkar) [reply examples/python-docker/Pipfile:11]: 0.0.12 is the latest version for cdktf as per the Pypi.

Comment (skorfmann) [reply examples/python-docker/Pipfile:11]: @anubhavmishra  meant `cdktf@next`, however we [don't release](https://pypi.org/project/cdktf/#history) pre releases for Python at the moment. I'm not even sure, if there's something comparable to `cdktf@next` in pypi.

However, I think for the examples we should stick to the monorepo references [as here](https://github.com/hashicorp/terraform-cdk/blob/b8aeded4cb58a7f367195d5ed8a4632588f43eca/examples/python-aws/Pipfile)

```
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[requires]
python_version = "3.7"

[packages]
cdktf = {path = "./../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```

Comment (abhishekamralkar) [reply examples/python-docker/Pipfile]: @skorfmann Pipfile updated to use the monorepo.

Comment (skorfmann) [reply examples/python-docker/Pipfile]: Cool, thanks 👍

**Comment by skorfmann** [comment]
Good from my point of view, @anubhavmishra ?

**Pull request merged**